apps/admin-x-activitypub/src/api/activitypub.test.ts

```diff
@@ -278,43 +278,6 @@ describe('ActivityPubAPI', function () {
 
             expect(actual).toEqual(expected);
         });
-
-        test('Returns an array when the items key is a single object', async function () {
```

Comment:
Removed this test case as Fedify now returns an array when there is a single object